src/main/java/Storage.java

```diff
@@ -0,0 +1,65 @@
+import java.io.*;
+import java.time.LocalDate;
+import java.util.ArrayList;
```

Comment:
I agree with shermz-lim, try to configure your IntelliJ so that it will remove this problem going forward

Reply:
Okay!